### ISSUE TITLE
ModuleNotFoundError: No module named 'certifi'

### ISSUE THREAD

**Issue by Ramblurr** [theY4Kman/python-pocketsmith-api#3]
### Problem

The module does not work out of the box in a fresh venv.

I think the issue is a missing dependency 

### Reproduce

```
python3 -m venv tmpvenv
source ./venv/bin/activate
pip install pocketsmith-api
```
Then execute the readme code:

```python
import pocketsmith
client = pocketsmith.PocketsmithClient('my-api-key')
client.users.get_me()
```

#### Result

```
Traceback (most recent call last):
  File "/var/home/ramblurr/src/finances/pocketsmith/test.py", line 2, in <module>
    import pocketsmith
  File "/var/home/ramblurr/src/finances/pocketsmith/venv/lib64/python3.9/site-packages/pocketsmith/__init__.py", line 21, in <module>
    from pocketsmith.api.accounts_api import AccountsApi
  File "/var/home/ramblurr/src/finances/pocketsmith/venv/lib64/python3.9/site-packages/pocketsmith/api/__init__.py", line 6, in <module>
    from pocketsmith.api.accounts_api import AccountsApi
  File "/var/home/ramblurr/src/finances/pocketsmith/venv/lib64/python3.9/site-packages/pocketsmith/api/accounts_api.py", line 21, in <module>
    from pocketsmith.api_client import ApiClient
  File "/var/home/ramblurr/src/finances/pocketsmith/venv/lib64/python3.9/site-packages/pocketsmith/api_client.py", line 30, in <module>
    from pocketsmith import rest
  File "/var/home/ramblurr/src/finances/pocketsmith/venv/lib64/python3.9/site-packages/pocketsmith/rest.py", line 22, in <module>
    import certifi
ModuleNotFoundError: No module named 'certifi'
```

## Workaround

```
pip install certifi
```

## Version:

```console
$ python3 --version
Python 3.9.1
```

```
 pocketsmith-api in ./venv/lib/python3.9/site-packages (1.0.0)
```

**Comment by theY4Kman** [comment]
oof, thanks for the report. that's embarrassing! i'll have this fixed shortly.

**Comment by theY4Kman** [comment]
Aight, fixed in [version 1.0.1](https://pypi.org/project/pocketsmith-api/1.0.1/). I also added some super basic tests around this, added a tox config to test on 3.6-3.9, and added a TravisCI config, so hopefully this class of mistakes is vanquished :P
